### PR TITLE
Clickhouse caching

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

**Comment by EDsCODE** [comment]
should wait after #2858 to test paths caching

File: posthog/celery.py [hunk 1]
```diff
@@ -65,13 +65,10 @@ def setup_periodic_tasks(sender, **kwargs):
...
 
-    if not is_ee_enabled():
-        sender.add_periodic_task(600, check_cached_items.s(), name="check dashboard items")
-    else:
-        # ee enabled scheduled tasks
-        sender.add_periodic_task(120, clickhouse_lag.s(), name="clickhouse table lag")
-        sender.add_periodic_task(120, clickhouse_row_count.s(), name="clickhouse events table row count")
-        sender.add_periodic_task(120, clickhouse_part_count.s(), name="clickhouse table parts count")
+    sender.add_periodic_task(90, check_cached_items.s(), name="check dashboard items")
+    sender.add_periodic_task(120, clickhouse_lag.s(), name="clickhouse table lag")
```
Comment (timgl) [inline posthog/celery.py:69]: these shouldn't be run for non-ee users

**Review by timgl — COMMENTED** [review]
Getting an error for paths on dashboard:
image

On the dashboard at least, it doesn't look like retention and paths are cached correctly (as they are still being loaded and don't appear as 'results' on the api/dashboard call)

File: posthog/celery.py (hunk 1 above)
Comment (EDsCODE) [reply posthog/celery.py]: yep accidentally reformed the if statement improperly

**Comment by EDsCODE** [comment]
The paths were broken separately and patched in #2858. Will check on retention and paths caching

**Comment by EDsCODE** [comment]
#2825 Solved the retention error and I patched the paths functions that kept getting called so now the dashboard item should be using cached data!

**Pull request merged**